src/test/groovy/StringTest.groovy

```diff
@@ -241,6 +241,11 @@ foo
 
         text = $/$$$$$//$
         assert text == '$$/'
+
+        //GROOVY-10024
+        def s1 = $/abc\/$
+        def s2 = $/def/$
+        assert s1.size() == 4 && s2.size() == 3
```

Comment:
I’d assert the expected values instead of the size.

Reply:
Does

```
def s1 = $/Failing string\\/$
```

With double backslash at the end work (as per the Jira example)?

Reply:
The double backslash case works - I'll adjust the test accordingly and also with expected values.

Reply:
Awesome as always Paul 😎👍